### DISCUSSION TITLE
Is this a collection of existing plugins, or new ones?

### DISCUSSION THREAD

**Discussion by MATOO-Dev** [folke/snacks.nvim#249] [Q&A]
For example, is snacks.lazygit the same as [kdheepak/lazygit.nvim](https://github.com/kdheepak/lazygit.nvim), snacks.notifier the same as [folke/noice.nvim](https://github.com/folke/noice.nvim), and so on?

Or another way to put it: should I install one over the other, or both? What are the differences?

**Comment by folke** [comment]
of course they're not the same. Check the docs
